provision-sm-md-lg/README.md

```diff
@@ -0,0 +1,70 @@
+# Provision VM Service Small Medium Large Example
+
+This example shows creating a provisioning dialog that modifies provisioning parameters.
+
+## Architecture
+
+- `provision-vm-small-med-large.asl`: This workflow is run for every vm provisioned and is responsible for modifying the vm size. This is the ManageIQ Service's Provisioning Entry Point.
+
+Set the ManageIQ provisioning service entry point to this script. This sample workflow is run for every provisioned Virtual Machine (VM), and modifies the VM size.
+
+## Add a Workflow Repository
+
+1. **Click Automation** > **Embedded Workflows** > **Repositories**
+2. **Click Configuration** > **Add new Repository**
+3. Provide the `Name` and `URL` for the repository:
+   - `Name`: Local name to identify this repository. Sample: `Example Workflows`
+   - `Url`: Git repository URL. Sample: `https://github.com/ManageIQ/workflows-examples`
+4. Click **Save**
+
+## Create a Provisioning Dialog
+
+1. Click **Automation** > **Embedded Automate** > **Customization**
+2. Click **Service Dialogs** > **All Dialogs**
+3. Click **Configuration** > **Add a new Dialog**
+4. Provide a `Name` and `Description` for the dialog:
+   - Name: Name used in the Service Catalog. Sample: `sm-med-lg dialog`
+   - Description: More information. Sample: `Dialog choosing vm size`
+5. Add a text box named `vm_name`.
+6. Add a Dropdown named `size`.
+7. Click the edit pencil icon for the "size" dropdown.
+8. Under **Options** > **Entries** enter the drop down values:
+   - Name=Large Value=large
+   - Name=Medium Value=medium
+   - Name=Small Value=small
+
+9. Click **Save**
+
+## Create a Service Catalog Item
+
+1. Click **Service** > **Catalogs**
+2. Click **Catalogs** > **All Catalogs**
+3. Click **Configuration** > **Add a New Catalog**
+4. Provide a `Name` and `Description`:
+   - `Name`: Name displayed in the catalog. Sample: `Provision Vm`
+   - `Description`: More information. Sample: `Provisioning Vm by size`
+5. Click **Save**
+6. Click **Catalog Items** > **All Catalog Items** > **Provisioning**
+7. Click **Configuration** > **Add a New Catalog Item**
+8. Choose `VmWare` as the value for `Catalog Item Type`
+9. Under the **Basic Information** Tab, provide the default VM values:
+   - `Name`: Name displayed in the catalog. Sample: `Provision Vm with Size`
+   - `Description`: More information. Sample: `Provision a vm choosing size`
+   - `Display in Catalog`: You always want this checked.
+   - `Catalog`: Organization concept. Sample: `My Company/Provisioning`
+   - `Dialog`: The dialog created above. Sample: `sm-med-lg dialog`.
+   - `Provisioning Entry Point`: `Embedded Workflow, provision-vm-small-med-large.asl`
```

Comment:
Same comment as Line 16 for all the fields.

Provisioning Entry Point: add description here. For example, `enter sample value here`.
This field is slightly unclear.